packages/react/src/components/Dialog/Dialog.stories.js

```diff
@@ -66,38 +69,53 @@ export const Modal = ({ open: _open, ...args }) => {
       <Button type="button" onClick={toggleDialog}>
         Toggle open
       </Button>
-      <Dialog {...args} open={open} onRequestClose={handleRequestClose}>
-        <DialogHeader>
-          {/* <DialogSubtitle></DialogSubtitle>
-          <DialogTitle></DialogTitle> */}
+      <Dialog
+        {...args}
+        open={open}
+        onRequestClose={handleRequestClose}
+        aria-label="Dialog Title">
+        <Dialog.Header>
+          <Dialog.Subtitle>Configure dialog settings</Dialog.Subtitle>
+          <Dialog.Title>Modal Dialog Example</Dialog.Title>
```

Comment:
Would this be better since we already have the Title as a "label"?

```suggestion
        aria-labelledby="title">
        <Dialog.Header>
          <Dialog.Subtitle>Configure dialog settings</Dialog.Subtitle>
          <Dialog.Title id="title">Modal Dialog Example</Dialog.Title>
```